### ISSUE TITLE
NeRF-Supervised Deep Stereo. (arXiv:2303.17603v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#25404]
NeRF-Supervised Deep Stereo. (arXiv:2303.17603v1 [cs.CV])<br>
https://ift.tt/vZ48htY<br>
<p>We introduce a novel framework for training deep stereo networks effortlessly and without any ground-truth. By leveraging state-of-the-art neural rendering solutions, we generate stereo training data from image sequences collected with a single handheld camera. On top of them, a NeRF-supervised training procedure is carried out, from which we exploit rendered stereo triplets to compensate for occlusions and depth maps as proxy labels. This results in stereo networks capable of predicting sharp and detailed disparity maps. Experimental results show that models trained under this regime yield a 30-40% improvement over existing self-supervised methods on the challenging Middlebury dataset, filling the gap to supervised models and, most times, outperforming them at zero-shot generalization.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
